lib/unit-ruby/util/api_resource.rb

```diff
@@ -101,6 +101,45 @@ def self.belongs_to(resource_name, class_name: nil)
       end
     end
 
+    def self.has_many(resource_name, class_name: nil)
+      class_name ||= resource_name.to_s.camelize
+
+      define_method(resource_name) do
+        singular_resource_name = resource_name.to_s.singularize.to_sym
+
+        # if there is only one of the resources, Unit will return the
+        # singular name of that relationship in their response.
+        # For example, if a deposit account has one customer, `customer`
+        # will be returned from Unit as a resource, whereas if there
+        # are more than one customer, then customers will be returned from Unit
+
+        # the below will translate these singular relationship
+        # into a plural relationship - ie `customer` -> `customers`
+
+        if relationships.keys.include?(singular_resource_name)
+
+          relationship_id = relationships.dig(singular_resource_name, :data, :id)
+
+          return nil unless relationship_id
```

Comment:
Since this is for the plural resource, should we return `[]` instead of nil?

(same on line L130)

Reply:
Good suggestion!